tests/integration/status-list-credential-module.test.js

```diff
@@ -0,0 +1,340 @@
+import { randomAsHex } from "@polkadot/util-crypto";
+
+import { DockAPI } from "../../src/index";
+
+import {
+  FullNodeEndpoint,
+  TestKeyringOpts,
+  TestAccountURI,
+} from "../test-constants";
+
+import { getHexIdentifierFromDID } from "../../src/utils/did";
+import { createNewDockDID } from "../../src/utils/did";
+import { OneOfPolicy } from "../../src/utils/revocation";
+import { registerNewDIDUsingPair } from "./helpers";
+import getKeyDoc from "../../src/utils/vc/helpers";
+import StatusList2021Credential from "../../src/status-list-credential/status-list2021-credential";
+
+const expectEqualCreds = (cred1, cred2) => {
+  expect(cred1).toEqual(cred2);
+  expect(cred1.toJSON()).toEqual(cred2.toJSON());
+  expect(StatusList2021Credential.fromJSON(cred1.toJSON())).toEqual(
+    StatusList2021Credential.fromJSON(cred2.toJSON())
+  );
+};
+const buildTest = process.env.DISABLE_STATUS_LIST_TESTS ? describe.skip : describe;
```

Comment:
`process.env.DISABLE_STATUS_LIST_TESTS` is a string so `buildTest` will be `describe.skip` even if its set to `false`.